tests/test_start_date.py

```diff
@@ -0,0 +1,26 @@
+from tap_tester.base_suite_tests.start_date_test import StartDateTest
```

Comment:
Per my comment above, looking at this kind of proves that we have too many imports on tests above that are not over-ridden and can be removed.  Here we just import the base suite test and the repo base file.